en/setup/dev_env_windows.md

```diff
@@ -1,122 +1,56 @@
 # Windows Installation Instructions
 
-There are a number of ways you can set up a Windows development toolchain for PX4. These allow you to build for NuttX/Pixhawk and jMAVSim simulator targets.
+There are a number of ways you can set up a Windows development toolchain for PX4. Depending on what option you choose these allow you to build for NuttX/Pixhawk and jMAVSim simulator targets.
 
-> **Warning** The Windows toolchains are not officially supported (and we discourage their use). They are unbearably slow during Firmware compilation and do not support new boards like Snapdragon Flight. They also cannot run the standard robotics software packages many developers use to prototype computer vision and navigation.
+> **Warning** The Windows toolchains are not officially supported (and we discourage their use). Depending on the option you choose they might be slow and not support all targets. Some options also cannot run the standard robotics software packages many developers use to prototype computer vision and navigation.
 
 <span></span>
-> **Tip** Before starting to develop on Windows, consider installing a dual-boot environment with [Ubuntu](http://ubuntu.com).
-
-
-## Native Toolchain
```

Comment:
I split up/copied all the existing instructions for the different options/methods without changing them to separate pages and hence new files. Only renaming the title of the method.

Reply:
I think the way you have done this split is excellent.